setup.sh

```diff
@@ -70,17 +92,60 @@ install_apt_package() {
     if [[ -n "${ENCLAVE_VERSION:-}" ]]; then
         # Install specified Enclave version
         info "Installing Enclave package (${ENCLAVE_VERSION})."
-        sudo apt-get install -yq "enclave=${ENCLAVE_VERSION}"
+        sudo apt install -yq "enclave=${ENCLAVE_VERSION}"
     else
         # Install latest Enclave
         info "Installing Enclave package (latest)."
-        sudo apt-get install -yq enclave
+        sudo apt install -yq enclave
     fi
 
     # Do not continue with rest of setup as deb handles all of it
     exit 0
```

Comment:
Do we need to change the apt path to function more like the RPM one now? It used to be that the deb file did the enrolment in-line, but we've stopped that now.